far2l/wrap.cpp

```diff
@@ -3614,7 +3614,7 @@ int WINAPI FarEditorControlA(int Command,void* Param)
 			if (Param)
 			{
 				oldfar::EditorSaveFile *oldsf = (oldfar::EditorSaveFile*) Param;
-				newsf.FileName=(oldsf->FileName)?AnsiToUnicode(oldsf->FileName):nullptr;
+				newsf.FileName=AnsiToUnicode(oldsf->FileName);
```

Comment:
this doesnt look as correct at all

Reply:
So you think these lines should stay the way they are?

Reply:
i think better to live without virtual of ~VTShell, or add empty virtual d-tor for all interfaces that it derived from.
Having virtual d-tor of class that derived from bases that have no virtual d-tor 'pushes' programmer to write buggy code, as he could assume that all.

Reply:
Oh, shame on me, I completely missed the fact that it has base classes. Removing this "fix".